app/assets/javascripts/modules/track-submit.js

```diff
@@ -0,0 +1,21 @@
+window.GOVUK.Modules = window.GOVUK.Modules || {};
```

Comment:
That makes sense. I was wondering whether this should be done in just the tests. It adds some protection against JS errors if this didn't run, for whatever reason: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/modules.js#L4

So I think it makes sense.